Build/Install-AcModule.ps1

```diff
@@ -0,0 +1,32 @@
+function Install-AcModule 
```

Comment:
What does Ac stand for?

Reply:
I cannot remember this is a function copied from the BuildAndDeploy module.

maybe it means avoid clobber?

Reply:
I don't think this entire function needs to exist at all, what is wrong with `Install-Module -name -requiredVersion`

Reply:
But, it isn't doing much harm and maybe someone knows something I don't. The inclusion of `-SkipPublisherCheck` is a tad worrying though.

Reply:
Let's raise an issue to investigate